scripts/scaffold/temporal/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine:latest
+
+RUN apk add curl
+RUN curl -sSf https://temporal.download/cli.sh | sh
```

Comment:
Why not https://hub.docker.com/r/temporalio/server ?

Reply:
The server image needs to be configured for development purpose. Also, it's heavier than just downloading the CLI for running a complete stack in development mode, with UI enabled and SQLite.